task.sql

```diff
@@ -3,31 +3,56 @@
 CREATE DATABASE ShopDB;
 USE ShopDB;
 
+CREATE TABLE Products (
+  ID INT,
+  ProductName VARCHAR(50),
+  PRIMARY KEY (ID)
+);
+
+CREATE TABLE Warehouses (
+  ID INT,
+  WarehouseName VARCHAR(50),
+  WarehouseAddress VARCHAR(50),
+  PRIMARY KEY (ID)
+);
+
 CREATE TABLE Countries (
-    ID INT,
-    Name VARCHAR(50),
-    PRIMARY KEY (ID)
+  ID INT,
+  Name VARCHAR(50),
+  PRIMARY KEY (ID)
 );
 
 CREATE TABLE ProductInventory (
-    ID INT,
-    ProductName VARCHAR(50),
-    WarehouseAmount INT,
-    WarehouseName VARCHAR(50),
-    WarehouseAddress VARCHAR(50), 
-    CountryID INT,
-	FOREIGN KEY (CountryID) REFERENCES Countries(ID) ON DELETE NO ACTION,
-    PRIMARY KEY (ID)
+  ID INT,
+  ProductID INT,
+  FOREIGN KEY (ProductID) REFERENCES Products (ID) ON DELETE NO ACTION,
+  WarehouseAmount INT,
+  WarehouseID INT,
+  FOREIGN KEY (WarehouseID) REFERENCES Warehouses (ID) ON DELETE NO ACTION,
+  CountryID INT,
+  FOREIGN KEY (CountryID) REFERENCES Countries (ID) ON DELETE NO ACTION,
```

Comment:
Need to remove `CountryID` column from this table

Reply:
Why? Could you explain?